src/components/HotkeyAnchor/HotkeyAnchor.tsx

```diff
@@ -61,13 +62,19 @@ export const HotkeyAnchor = () => {
     enabled: state.hotkeysAreActive && isAdmin,
   };
 
+  function dispatchHotkeyNotification(title: string) {
+    if (state.hotkeyNotificationsEnabled) {
+      Toast.info({title, autoClose: TOAST_TIMER_SHORT});
+    }
+  }
```

Comment:
This should be an arrow function, like the others
```suggestion
  const dispatchHotkeyNotification = (title: string) => {
    if (state.hotkeyNotificationsEnabled) {
      Toast.info({title, autoClose: TOAST_TIMER_SHORT});
    }
  };
```